File: Лабораторная работа №2/C#/MyString.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace StringLibrary
{
    public class MyString
    {
		public char[] arrgs;
		public int count = 0;
		public MyString(char[] line = null)
		{
			arrgs = line;
		}
		public int СharCount(MyString arr, char c)
		{
			int k = 0;
			for (int j = 0; j < arr.count; j++)
			{
				if (arr.arrgs[j] == c)
				{
					k += 1;
				}
			}
			return k;
		}
		public void outputString()
		{
			for (int i = 0; i < arrgs.Length; i++)
			{
				if (arrgs[i] == default(char))
					break;
				Console.Write(arrgs[i]);
			}
		}
		public void GetStr(char[] arr)
		{
			arrgs = arr;
			for (int i = 0; i < 1000; i++)
			{
				if (arr[i] == default(char))
					break;
				count++;
			}
		}
		~MyString()
		{
		}
	}
}

File: Лабораторна робота №2/Лабораторна робота №2/MyString.cpp
#pragma once
#include"MyString.h"

int MyString::ÑharCount(MyString arr,char c)
{
	int k = 0;
		for (int j = 0; j <arr.count; j++)
		{
			if (arr.arrgs[j] == c)
			{
				k += 1;
			}
	}
	return k;
}
void MyString::GetStr(std::vector<char>arr)
{	
	arrgs = arr;
	count = arr.size();
}
int MyString::Size(MyString arr,int j)
{
	char c = ' ';
	int k = 0;
	for(int i=0;i<count;i++)
	{
		c = arr.arrgs[i];
		k++;
	}
	return k;
}
MyString::~MyString()
{
	arrgs.clear();
}
